src/test/java/org/cactoos/http/HtTimedWireTest.java

```diff
@@ -60,20 +59,20 @@ public void worksFine() throws Exception {
         );
     }
 
-    @Ignore("see BlockingSocketServer todo for #87")
     // @checkstyle MagicNumberCheck (1 line)
     @Test(expected = TimeoutException.class, timeout = 1000)
+    @SuppressWarnings("PMD.AvoidUsingHardCodedIP")
     public void failsAfterTimeout() throws Exception {
-        // @checkstyle MagicNumberCheck (1 line)
+        // @checkstyle MagicNumberCheck (8 lines)
         final long timeout = 100;
-        try (BlockingSocketServer server = new BlockingSocketServer()) {
-            new HtTimedWire(
-                new HtWire(
-                    server.address().getHostAddress(),
-                    server.port()
-                ),
-                timeout
-            ).send(new InputOf("unused"));
-        }
+        new HtTimedWire(
+            new HtWire(
+                // non-routable IP address artificially creates a timeout error
+                // see: https://stackoverflow.com/a/904609/3456163
+                "10.255.255.1",
```

Comment:
@iakunin Let's put this comment in the method javadoc, avoiding these comment inside the method

Reply:
@paulodamaso done